app/routes/news._index.mdx

```diff
@@ -26,6 +26,26 @@ import { Anchor } from '~/components/Anchor'
 
 ## 2024
 
+    <CollectionItem
+      className="maxw-none"
+      variantComponent={
+        <CollectionCalendarDate datetime={'November 8, 2024'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>VTP Migration Survey and Legacy Circular Email Retirement</Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+        - **VOEvent Transport Protocol (VTP) Migration Survey**
```

Comment:
Make this a level 4 heading (`####`) rather than a bullet point. Same for the second heading.